src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h

```diff
@@ -119,12 +119,190 @@ typedef enum {
 typedef enum {
     /* 0 */ PAUSE_BG_PRERENDER_OFF,
     /* 1 */ PAUSE_BG_PRERENDER_SETUP, // The current frame is only drawn for the purpose of serving as the pause background.
-    /* 2 */ PAUSE_BG_PRERENDER_PROCESS, // The previous frame was PAUSE_BG_PRERENDER_DRAW, now apply prerender filters.
+    /* 2 */ PAUSE_BG_PRERENDER_PROCESS, // The previous frame was PAUSE_BG_PRERENDER_SETUP, now apply prerender filters.
     /* 3 */ PAUSE_BG_PRERENDER_DONE, // The pause background is ready to be used.
     /* 4 */ PAUSE_BG_PRERENDER_UNK4,
     /* 5 */ PAUSE_BG_PRERENDER_MAX
 } PauseBgPreRenderState;
 
+typedef enum {
+    /* 0 */ VTX_PAGE_MASK,
+    /* 1 */ VTX_PAGE_ITEM,
+    /* 2 */ VTX_PAGE_MAP_DUNGEON,
+    /* 3 */ VTX_PAGE_QUEST,
+    /* 4 */ VTX_PAGE_MAP_WORLD,
+    /* 5 */ VTX_PAGE_PROMPT,
+    /* 6 */ VTX_PAGE_MAX
+} VtxPage;
+
+#define VTX_PAGE_MASK_QUADS 0        // VTX_PAGE_MASK
+#define VTX_PAGE_ITEM_QUADS 0        // VTX_PAGE_ITEM
+#define VTX_PAGE_MAP_DUNGEON_QUADS 6 // VTX_PAGE_MAP_DUNGEON
+#define VTX_PAGE_QUEST_QUADS 0       // VTX_PAGE_QUEST
+#define VTX_PAGE_MAP_WORLD_QUADS 36  // VTX_PAGE_MAP_WORLD
+#define VTX_PAGE_SAVE_QUADS 5        // VTX_PAGE_PROMPT
+
+#define PAGE_BG_COLS 3
+#define PAGE_BG_ROWS 5
+#define PAGE_BG_QUADS (PAGE_BG_COLS * PAGE_BG_ROWS)
+#define PAGE_BG_QUAD_WIDTH 80
+#define PAGE_BG_QUAD_HEIGHT 32
+#define PAGE_BG_QUAD_TEX_WIDTH 80
+#define PAGE_BG_QUAD_TEX_HEIGHT 32
+
+// === ITEM === //
+
+#define ITEM_GRID_ROWS 4
+#define ITEM_GRID_COLS 6
+#define ITEM_GRID_CELL_WIDTH 32
+#define ITEM_GRID_CELL_HEIGHT 32
+#define ITEM_GRID_QUAD_MARGIN 2
+#define ITEM_GRID_QUAD_WIDTH (ITEM_GRID_CELL_WIDTH - (2 * ITEM_GRID_QUAD_MARGIN))
+#define ITEM_GRID_QUAD_HEIGHT (ITEM_GRID_CELL_HEIGHT - (2 * ITEM_GRID_QUAD_MARGIN))
+#define ITEM_GRID_QUAD_TEX_SIZE 32 // both width and height
+#define ITEM_GRID_QUAD_ENLARGE_OFFSET 2
+
+#define ITEM_GRID_SELECTED_QUAD_MARGIN (-2)
+#define ITEM_GRID_SELECTED_QUAD_WIDTH (ITEM_GRID_QUAD_WIDTH - (2 * ITEM_GRID_SELECTED_QUAD_MARGIN))
+#define ITEM_GRID_SELECTED_QUAD_HEIGHT (ITEM_GRID_QUAD_HEIGHT - (2 * ITEM_GRID_SELECTED_QUAD_MARGIN))
+#define ITEM_GRID_SELECTED_QUAD_TEX_SIZE 32 // both width and height
+
+#define ITEM_NUM_SLOTS (ITEM_GRID_ROWS * ITEM_GRID_COLS)
+
+typedef enum {
+    // 0..59 are the 15 background textures
```

Comment:
I think these should all be made consistent and always refer to quads if possible, it's rather confusing atm.

Reply:
Perhaps I'll defer this to the follow-up vtx PR?